Components/avatar.css

```diff
@@ -0,0 +1,58 @@
+.avatar-default{
+    width: 4rem;
+    height :4rem;
+    border-radius: 50%;
+    overflow: hidden;
+
+}
+
+.avatar-sm{
+    width: 2.5rem ;
+    height: 2.5rem ;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.avatar-lg{
+    width: 6rem;
+    height: 6rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
```

Comment:
combine common properties

Reply:
done ,combined border radius and overflow into avatar-round done